### DISCUSSION TITLE
<Is it possible to autocomplete citations in Jupyterlab? >

### DISCUSSION THREAD

**Discussion by lucasfindslinks** [quarto-dev/quarto-cli#6674] [Q&A]
### Description

Similar to what has been asked[ here for Rstudio ](https://github.com/quarto-dev/quarto-cli/discussions/2441) , is there a current way to have Overleaf style citation autocomplete (starting from the @ operator in markdown)  while using Jupyterlab?  

 I'm working on a scientific paper on JupyterLab with Quarto. My citations are in Bibtex format, defined in the YAML header,  and are being output correctly in the html/pdf. I saw that its a possibility for VSCode using the plugin or for RStudio, but nothing specifically for Jupyter .

Thanks!

**Comment by cscheid** [comment]
I don't think it is. Quarto's autocomplete in VS Code and RStudio is more sophisticated than what's available in JupyterLab. I know our extension does some work, but I don't think it resolves references (like VS Code's quarto extension and RStudio's native support)
